hw/ip/aes/pre_dv/aes_sbox_lec/aes_sbox_lec.py

```diff
@@ -26,13 +26,19 @@ def replace_module_name(file_name, string_search, string_replace):
 # List all S-Box reference implementation + AES package
 impl_gold = 'aes_sbox_lut'
 file_pkg = 'aes_pkg.sv'
+file_pkg_canright = 'aes_sbox_canright_pkg'
 
 # Detect all S-Box implementations to check
 impl_list = glob.glob(rtl_path + 'aes_sbox_*.sv')
 impl_list = [
     impl_dut.replace(rtl_path, '').replace('.sv', '') for impl_dut in impl_list
 ]
 impl_list.remove(impl_gold)
+impl_list.remove(file_pkg_canright)
+file_pkg_canright = file_pkg_canright + '.sv'
+
+# Remove masked implementations, we can't do LEC against the LUT for those.
```

Comment:
Would it be possible to LEC the masked versions with the mask set to zero?  I might not understand the LEC issue here though.

Reply:
I thought about that, too. But Yosys (what I am using for LEC) seems not to support forcing inputs. I think the way to go is to add a LEC wrapper that drives all mask inputs with 0. I will look into that.

Reply:
I have added a wrapper to perform LEC against the LUT-based implementation. I am resolving this one.